modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -9,7 +9,13 @@ For an equivalent to clustering in CloudHub, see xref:runtime-manager::cloudhub-
 
 Mule Enterprise Edition supports scalable clustering to provide high availability (HA) for applications.
 
-A cluster is a set of Mule runtime engines that acts as a unit. In other words, a cluster is a virtual server composed of multiple nodes (Mule runtime engines). The nodes in a cluster communicate and share information through a distributed shared memory grid. This means that the data is replicated across memory in different machines.
+A cluster is a set of Mule runtime engines that acts as a unit. In other words, a cluster is a virtual server composed of multiple nodes (Mule runtime engines). The nodes in a cluster communicate and share Object Store and VM queue data through a distributed shared memory grid. This means that the data is replicated across memory in different machines. 
+
+This shared memory grid, replicated state applies to: 
+
+* persistent and non-persistent Object stores 
```

Comment:
Object stores -> Object Store instances ?

Reply:
Don't think we really use the term "instances" elsewhere - the Object Store connector for example just says "Object store" for stuff like "default object store". So instances I think is redundant.